openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -100,6 +100,13 @@ def to_representation(self, instance):
         return super().to_representation(instance)
 
     def get_tags_count(self, instance):
+        """
+        Return the "tags_count" annotation if present.
+
+        Or just count the taxonomy's tags.
+        """
+        if hasattr(instance, 'tags_count'):
+            return instance.tags_count
```

Comment:
Did this help reduce the query count? The reason I ask is because on the commit that this was added in, the query counts in the tests did not change, so I was just wondering what effect this may have had.

Reply:
The query counts dropped here when this branch was used: https://github.com/openedx/edx-platform/pull/34200/commits/016f2a54351257a66d0c90abe2e7079e7940ccb9